bento_sdk.routing.yml

```
@@ -6,4 +6,14 @@ bento_sdk.settings:
\ No newline at end of file
+    _custom_access: '\Drupal\bento_sdk\Form\BentoSettingsForm::access'
+
+bento_sdk.test_email:
+  path: '/admin/config/bento/test-email'
+  defaults:
+    _controller: '\Drupal\bento_sdk\Controller\TestEmailController::sendTestEmail'
+  methods: [POST]
+  requirements:
+    _permission: 'administer bento sdk'
+  options:
+    _csrf_token: FALSE
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)